tests/robot-tests/tests/general_public/table_tool_absence_by_characteristic.robot

```diff
@@ -129,22 +128,29 @@ Reorder Gender to be column group
     user opens details dropdown    Re-order table headers
     # Column group needs to be inside the viewport
     user scrolls to element    xpath://button[text()="Re-order table"]
-    user sets focus to element    xpath://legend[text()="Row group 1"]/../../..
+    user clicks button    Re-order row groups
+    user clicks button    Move Characteristic group to columns
+
+Reorder Gender to be first of the column groups
+    user clicks button    Re-order column groups
+    user sets focus to element    xpath://legend[text()="Characteristic"]/../../..
     user presses keys    ${SPACE}
-    user presses keys    ARROW_DOWN
     user presses keys    ARROW_LEFT
     user presses keys    ${SPACE}
 
 Reorder Gender male to be second
+    ${columnGroups}=    user gets testid element    columnGroups
+    user clicks button    Re-order items    columnGroups
     # The /../.. to get to a focusable element
-    user sets focus to element    xpath://strong[text()="Gender male"]/../..
+    user sets focus to element    xpath://span[text()="Gender male"]/../..
```

Comment:
Would consider using `*` instead of `span`. If there are potentially other candidate elements, we should constrain the selector using the `parent` argument:

```robot
user sets focus to element   xpath://*[text()="..."]/../..   testid:sortableListGroup-columns
```
I've used a testid selector here (may need to add one to the parent container), but it could be whatever selector.

Same applies for the other keywords below